utils/spdx/src/main/resources/declared-license-mapping.yml

```diff
@@ -405,6 +405,7 @@
 "The GNU Lesser General Public License, Version 2.1": LGPL-2.1-only
 "The GNU Lesser General Public License, Version 3.0": LGPL-3.0-only
 "The Go license": BSD-3-Clause
+"The Go License": BSD-3-Clause AND LicenseRef-scancode-google-patent-license-golang
```

Comment:
Looking into the POM file it has:
```
<name>Go License</name>
<url>https://golang.org/LICENSE</url>
```

I'd interpret this as "Go License" referring to "https://golang.org/LICENSE". So, I believe the value should be "BSD-3-Clause" only, not "BSD-3-Clause AND LicenseRef-scancode-google-patent-license-golang". Put I'd like to have other opinions, e.g. @oss-review-toolkit/core-devs ?

Reply:
I agree that it should only map to `BSD-3-Clause`, also so that the mappings for `"The Go license"` and `"The Go License"` match (it would be weird otherwise).